### PR TITLE
v1.4.8.0 release

### PR DESCRIPTION
**Pull request by iulianpascalau**
- rc/v1.4.0 compatible configs
- agreed activation epochs is set to 938 which should happen on 2023.02.23 ~15:30

### CODE REVIEW

**Review by sasurobert — DISMISSED** [review]

**Review by andreibancioiu — DISMISSED** [review]

**Review by bogdan-rosianu — DISMISSED** [review]

File: config.toml [hunk 1]
```diff
@@ -920,6 +919,7 @@
     PeerAuthenticationTimeBetweenSendsWhenErrorInSec = 60   # 1min
     PeerAuthenticationThresholdBetweenSends          = 0.1  # 10%
     HeartbeatTimeBetweenSendsInSec                   = 300  # 5min
+    HeartbeatTimeBetweenSendsDuringBootstrapInSec    = 300  # 5min
```
Comment (sstanculeanu) [inline config.toml:922]:
```suggestion
    HeartbeatTimeBetweenSendsDuringBootstrapInSec    = 60   # 1min
```
for a better feedback on the real time trie sync status

Comment (iulianpascalau) [reply config.toml:922]: It will spam the network. Decided with @AdoAdoAdo to be left to 5 minutes

Comment (iulianpascalau) [reply config.toml:922]: Anyway, anyone can manually alter it in prefs.toml :)

Comment (sstanculeanu) [reply config.toml:922]: :+1:

**Review by sstanculeanu — APPROVED** [review]

**Review by raduchis — DISMISSED** [review]

**Review by AdoAdoAdo — DISMISSED** [review]

File: docker/Dockerfile [hunk 2]
```diff
@@ -13,7 +13,7 @@ WORKDIR /go/mx-chain-go
 # MultiversX node
 WORKDIR /go/mx-chain-go/cmd/node
 RUN go build -i -v -ldflags="-X main.appVersion=$(git --git-dir /config/.git describe --tags --long --dirty)"
-RUN cp /go/pkg/mod/github.com/!elrond!network/arwen-wasm-vm@$(cat /go/mx-chain-go/go.mod | grep arwen-wasm-vm | sed 's/.* //' | tail -n 1)/wasmer/libwasmer_linux_amd64.so /lib/libwasmer_linux_amd64.so
+RUN cp /go/pkg/mod/github.com/multiversx/$(cat /go/mx-chain-go/go.mod | grep mx-chain-vm-v | sort -n | tail -n -1| awk -F '/' '{print$3}'| sed 's/ /@/g')/wasmer/libwasmer_linux_amd64.so /lib/libwasmer_linux_amd64.so || :
```
[the remark this answers is not in the document]
Comment (iulianpascalau) [reply docker/Dockerfile]: fixed

**Review by andreibancioiu — APPROVED** [review]

**Review by miiu96 — APPROVED** [review]

**Review by sstanculeanu — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by raduchis — APPROVED** [review]

**Pull request merged**